app/helpers/responses_helper.rb

```diff
@@ -2,11 +2,11 @@ module ResponsesHelper
   def display_value_for_number(response)
     return if !response.has_number?
     if response.has_currency?
-      "#{prefix(response)}#{response.number} #{postfix(response)}"
+      "#{prefix(response)}#{format_number(response.number)} #{postfix(response)}"
```

Comment:
any particular reason not to just use `number_with_delimiter` directly?

Reply:
Only if we need to support locales that use `.` in place of `,`. We'd just need to pass in the additional option. but I like your approach to just use `number_with_delimiter` directly.